deploy/nl/embeddings.yaml

```diff
@@ -2,19 +2,29 @@ medium_ft:
   store: MEMORY
   embeddings: gs://datcom-nl-models/embeddings_medium_2024_03_14_16_38_53.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
   model: ft_final_v20230717230459.all-MiniLM-L6-v2
+  model_type: LOCAL
```

Comment:
Custom DC also generates the YAML (@keyurva), so could you also update:

https://github.com/datacommonsorg/website/blob/d61af7aff064b0c92db702473c0f7113e5e17dda/tools/nl/embeddings/build_custom_dc_embeddings.py#L140-L150
https://github.com/datacommonsorg/website/blob/d61af7aff064b0c92db702473c0f7113e5e17dda/tools/nl/embeddings/testdata/custom_dc/expected/custom_embeddings.yaml#L4

Reply:
done

Reply:
Will it cause issues if `MODEL_TYPE` is not present? If it does, can we prevent it? In general, for custom DCs, we need to preserve backward compatibility.